django/researchdata/templates/researchdata/transcribe-fail.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block main %}
+<div class="container">
+
+    <section>
+        <h2>Error in Submission</h2>
+        <p>
+            There was an error when trying to submit your transcription. Please press back in your browser to return to your transcription. You may wish to copy and paste the content of your transcription into another place (e.g. into a Word document) to keep it safe. Please then try to resubmit the form.
```

Comment:
Can line breaks be put into the <p> tags? Just wondering if these long lines might trigger a linter at some point.